### DISCUSSION TITLE
How to create from scratch the amd64-18.04-xbb-v3.4

### DISCUSSION THREAD

**Discussion by bvernoux** [xpack-dev-tools/riscv-none-elf-gcc-xpack#8] [Q&A]
How to create from scratch the amd64-18.04-xbb-v3.4 image ?

- I would like to avoid using Docker & the "bash ${HOME}/Work/riscv-none-elf-gcc-xpack.git/scripts/helper/build.sh preload-images" and instead start from a native Ubuntu 18.04 install with all dependencies to have same image and build riscv-none-elf-gcc-xpack

@ilg-ul Could you provide the full process how to create that amd64-18.04-xbb-v3.4 ?

**Comment by ilg-ul** [comment]
With the new version of the build scripts you no longer need that docker image, you can run the build natively on any Linux with a standard set of development tools.

For a preview, you can take a look at the xpack-develop branch. 

A final release will hopefully be available in about one week.
